mypy.ini

```diff
@@ -15,3 +15,6 @@ follow_imports = silent
 
 [mypy-grpc]
 ignore_missing_imports = True
+
+[mypy-requests.*]
+ignore_missing_imports = True
```

Comment:
Just for my own understanding, why is this needed?

Reply:
fixed it 🙂